web/app/public/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="fr">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Manala - configuration</title>
+</head>
+<body>
+    <div id="app" class="app"></div>
```

Comment:
```suggestion
    <div id="app" class="app">Manala Web UI</div>
```

Sert de placeholder pour l'app